src/seedu/addressbook/data/person/Printable.java

```diff
@@ -1,7 +1,10 @@
 package seedu.addressbook.data.person;
 
 public interface Printable {
-    default void print() {
-        System.out.println("Person is here");
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    default String getPrintableString() {
```

Comment:
should this have a default implementation? think about what happens when a class that implements this interface but did not override this method